hashicorp/dev-portal
# feat: add gradient border

> **Note**: 🏗 This PR targets the Certifications assembly branch. See #1447 for details.

## 🗒️ What

This PR implements a gradient border for `GradientCard` component. The `GradientCard` component is used on both the landing page, and on individual certifications pages.

This PR takes the approach to gradient borders described in <https://css-tricks.com/gradient-borders-in-css/>.

## 📸 Design Screenshots

| Page | Before | After |
| - | - | - |
| Landing | <img width="936" alt="landing-before" src="https://user-images.githubusercontent.com/4624598/208502543-126fec4a-f7e9-4e9d-9a90-1d3f8f536cdd.png"> | <img width="921" alt="landing-after" src="https://user-images.githubusercontent.com/4624598/208502553-9edc47b1-fa3b-48bd-a30c-a3118c80f7c0.png"> |
| Program page | <img width="923" alt="program-page-before" src="https://user-images.githubusercontent.com/4624598/208502534-1ca7f666-8ff6-4157-8fb6-4e86adee2f78.png"> | <img width="928" alt="program-page-after" src="https://user-images.githubusercontent.com/4624598/208502537-0b118229-7a27-40d0-923e-afbb5a7247e5.png"> |

## 🔗 Preview links

- [/certifications][/certifications]
- [/certifications/infrastructure-automation][/certifications/infrastructure-automation]
- [/certifications/networking-automation][/certifications/networking-automation]
- [/certifications/security-automation][/certifications/security-automation]

## 🧪 Testing

- [ ] Visit each of the preview links listed above
    - Ensure the `GradientCard` has a gradient background
    - Ensure each gradient card element appears as expected

[preview]: https://dev-portal-git-zscert-gradient-border-hashicorp.vercel.app/
[/certifications]: https://dev-portal-git-zscert-gradient-border-hashicorp.vercel.app/certifications
[/certifications/infrastructure-automation]: https://dev-portal-git-zscert-gradient-border-hashicorp.vercel.app/certifications/infrastructure-automation
[/certifications/networking-automation]: https://dev-portal-git-zscert-gradient-border-hashicorp.vercel.app/certifications/networking-automation
[/certifications/security-automation]: https://dev-portal-git-zscert-gradient-border-hashicorp.vercel.app/certifications/security-automation
